src/WebJobs.Script.NuGet/WebJobs.Script.nuspec

```diff
@@ -16,12 +16,12 @@
     <dependencies>
       <dependency id="Microsoft.Azure.WebJobs" version="2.0.0-alpha-10331" />
```

Comment:
The 10298 extensions package depends on a higher version of the core SDK assemblies. See myget